src/Components/Popover/Base/Popover.tsx

```diff
@@ -55,13 +59,13 @@ export interface PopoverProps extends StandardFunctionProps {
   enableDefaultStyles?: boolean
 }
 
-export type PopoverRef = HTMLSpanElement
+export type PopoverRef = PopoverDialogRef
 
 interface CustomMouseEvent extends MouseEvent {
   relatedTarget: Element
 }
 
-let uniquePortalID: string = ''
+const popoverPortalName = 'popover'
```

Comment:
Is this to be consistent for all shared for all popovers globally?

Reply:
yarrrr